lib/board.rb

```diff
@@ -38,22 +38,23 @@ def valid_placement?(ship, coordinates)
   def place(ship, coordinates)
     if valid_placement?(ship, coordinates) #moved this up, if valid_placement?(ship, coordinates) then allow rest of method to run
        coordinates.each do |coordinate|#each method is being called on the coordinates array(from cell class)
-         @cell = @cells[coordinate] #creating local variable for the cell object(@cells[coordinate])
-        @cell.place_ship(ship) #call place_ship method(from cell class) on variable we created for cell object, pass in ship argument
+         cell = @cells[coordinate] #creating local variable for the cell object(@cells[coordinate])
```

Comment:
I think the @cell was referring to a cell object in our cell spec file @cell = Cell.new('B4')